_backend/config.example.php

```diff
@@ -24,6 +24,9 @@
     'stripe_sk' => 'sk_test_hoigesrjgoisrhgilgjrsfjs',
     'stripe_pk' => 'pk_test_hoigesrjgoisrhgilgjrsfjs',
 
+    'previous_stripe_sk' => 'sk_test_hoigesrjgoisrhgilgjrsfjs',
+    'previous_stripe_pk' => 'pk_test_hoigesrjgoisrhgilgjrsfjs',
```

Comment:
We don't need the previous `pk`

Reply:
Do we want to include it anyway in case it is needed in the future? I assume these are from the same page in a Stripe account.

Reply:
It's from the same page, but the `pk` is used for the client to generate new charge tokens. Considering we should _never_ create a new charge on the old account, I think it's safe to assume we will _never_ need to use it.